generator/graphql-kotlin-schema-generator/src/test/kotlin/com/expediagroup/graphql/generator/internal/types/GenerateUnionTest.kt

```diff
@@ -94,4 +98,15 @@ internal class GenerateUnionTest : TypeTestHelper() {
         assertEquals(1, result.directives.size)
         assertEquals("simpleDirective", result.directives.first().name)
     }
+
+    @Test
```

Comment:
should also add a test when the annotation has an empty `possibleTypes` along with `name`